src/supervised/scaledloss.jl

```diff
@@ -1,4 +1,4 @@
-for KIND in (:MarginLoss, :DistanceLoss)
+for KIND in (:MarginLoss, :DistanceLoss, :SupervisedLoss)
```

Comment:
ZeroOneLoss is not really a practical loss. It is the "ideal" classification goal of sorts. It is mostly in the package for completeness sake and so that users can plot it. AFAIK it is not a proper MarginLoss. Looking at the code now I don't remember why not. Need to look in the book again